specs/endpoint-security.spec.yml

```diff
@@ -60,6 +62,7 @@ inputs:
         - condition: ${runtime.user.root} == false
           message: "Elastic Agent must be running as Administrator or SYSTEM"
     service:
+      cport: 6788
```

Comment:
please include increased timeout from PR #1286

Reply:
Removed the timeout for Stop from V2.
Based on conversation with Endpoint team earlier. The STOPPING state issuing should be the best effort attempt and followed with uninstall. STOPPING is effectively NOOP for the normal Endpoint installation, since the uninstall will take care of stopping and uninstalling the Endpoint service. It's only needed for k8s installation support, so the endpoint can be "deactivated" in k8s environment.

Now when the Endpoint removed from the policy the following sequence happening:
1. If Endpoint has not checked-in yet, the Agent awaits for check-in up to the check-in timeout duration
2. The Agent sends STOPPING as expected state to the Endpoint if Endpoint checked-in
3. The Agent uninstalls Endpoint, logs the error if there was an error.
4. The Agent sets the service runtime into STOPPED state always. This avoids leaving the Agent in unrecoverable failed state.